pkg/api/v1/pod/util_test.go

```diff
@@ -316,38 +316,113 @@ func TestPodSecrets(t *testing.T) {
 	}
 }
 
-// collectSecretPaths traverses the object, computing all the struct paths that lead to fields with "secret" in the name.
-func collectSecretPaths(t *testing.T, path *field.Path, name string, tp reflect.Type) sets.String {
-	secretPaths := sets.NewString()
+// collectResourcePaths traverses the object, computing all the struct paths that lead to fields with resourcename in the name.
+func collectResourcePaths(t *testing.T, resourcename string, path *field.Path, name string, tp reflect.Type) sets.String {
```

Comment:
this function and package should stay in sync with `pkg/api/pod/util.go`

Reply:
They is same.  
The function in pkg/api/pod/util.go have been modified in pr https://github.com/kubernetes/kubernetes/pull/55465.